package/python-kiwi-spec-template

```diff
@@ -80,6 +80,8 @@ and cloud systems like Xen, KVM, VMware, EC2 and more.
 %package -n python%{python3_pkgversion}-kiwi
 Summary:        KIWI - Appliance Builder Next Generation
 Group:          Development/Languages/Python
+Obsoletes:      python2-kiwi
+Conflicts:      kiwi-man-pages < 9.20.5
```

Comment:
I am wondering if it wouldn't be more generic to do something like `Conflicts: kiwi-man-pages < %{version}`. This way, in addition of solving the former issue, we force some alignment between kiwi-man-pages package and python3-kiwi which is, IMHO, not bad.

Reply:
yes better